e2e/app/page/cohort-participants-group.ts

```diff
@@ -104,7 +104,7 @@ export default class CohortParticipantsGroup {
   }
 
   async getGroupCount(): Promise<string> {
-    return waitForNumericalString(this.page, this.getGroupCountXpath());
+    return waitForNumericalString(this.page, this.getGroupCountXpath(), 60000);
```

Comment:
Group Count is calculated in bigquery when Finish button is clicked. It could take some time.